tests/integ/modin/io/test_read_csv.py

```diff
@@ -138,21 +135,6 @@ def test_read_csv_names_overwrite_header(resources_path):
 @pytest.mark.parametrize(
     "names, error_msg, expected_query_count",
     [
-        (
-            ("c1", "c2", "c3", "c4"),
```

Comment:
should we add these cases elsewhere?

Reply:
No, these work in native pandas. Maybe they didn't work in 1.5 but they work in 2.2